fedimint-core/src/api.rs

```diff
@@ -1099,7 +1100,11 @@ mod tests {
         };
 
         let bech32 = connect.to_string();
+        let json = serde_json::to_string(&connect).unwrap();
+        assert_eq!(json, format!("\"{bech32}\""));
```

Comment:
Wait. I don't get it. What is going on here? json formatting is `\"{bech32}\"`? How does this ... what?

Reply:
\ is just escape here. so format is `"{bech32}"`

Reply:
Thank you. I need more coffee.